src/HPWH.cc

```diff
@@ -63,6 +63,10 @@ const double HPWH::MAXOUTLET_R410A = F_TO_C(140.);
 const double HPWH::MAXOUTLET_R744 = F_TO_C(190.);
 const double HPWH::MINSINGLEPASSLIFT = dF_TO_dC(15.);
 
+double makeC(const double T_F_or_C,const HPWH::UNITS units,const bool absolute){
```

Comment:
This can probably be inline, no?

The name is also not very descriptive. Maybe `convertTempToC`?